frontend/src/scenes/web-analytics/SessionDebugger/sessionAttributionExplorerLogic.ts

```diff
@@ -73,27 +91,33 @@ ORDER BY 8 DESC
 
     actionToUrl(({ values }) => {
         const stateToUrl = (): [string, Record<string, string>] => {
-            const { filters } = values
+            const { properties, dateRange } = values
 
             const urlParams = {}
```

Comment:
not blocking nit we're replacing all existing query and hash params and here rather than adding/editing them
it's fine really but neater to let things we don't need to change here stay